scripts/bump_version.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Setting -x is absolutely forbidden as it could leak the GitHub token.
+set -uo pipefail
+
+
+GITHUB_MAIL="${GITHUB_MAIL:-prometheus-team@googlegroups.com}"
+GITHUB_USER="${GITHUB_USER:-prombot}"
+
+# GITHUB_TOKEN required scope: repo.repo_public
+GITHUB_TOKEN="${GITHUB_TOKEN:-}"
+if [ -z "${GITHUB_TOKEN}" ]; then
+	echo -e "\e[31mGitHub token (GITHUB_TOKEN) not set. Terminating.\e[0m"
+	exit 1
+fi
+
+go run ./cmd/builder-bumper
+
```

Comment:
Let's check `$?` for failed go run.

Reply:
Added `set -e` at the beginning of the script.

Reply:
I prefer to avoid `set -e` as it causes failures to be silent.